apps/concept_items/resource.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8; -*-
+#
+# This file is part of Superdesk.
+#
+# Copyright 2017 Sourcefabric z.u. and contributors.
+#
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+
+from superdesk.resource import Resource
+
+
+CONCEPT_ITEMS_PRIVILEGE = 'concept_items'
+
+
+class ConceptItemResource(Resource):
+    schema = {
+        'first_name': {
```

Comment:
I think we should have just `name` otherwise this doesn't work for other types of concepts